File: src/org/usfirst/frc/team1678/robot/ManualControls.java
package org.usfirst.frc.team1678.robot;

import edu.wpi.first.wpilibj.*;s

public class ManualControls {
	static Joystick control;
	void run()
	{
		RobotParts.yAxisMotor1.set(control.getRawAxis('Y'));
		RobotParts.yAxisMotor2.set(control.getRawAxis('Y'));
		
		RobotParts.xAxisMotor.set(control.getRawAxis('X'));
		
		if(control.getRawButton(1))
		{//pemis
			RobotParts.grabberMotor1.set(.3);
			RobotParts.grabberMotor2.set(.3);
			RobotParts.grabberMotor3.set(.3);
		}
		else
		{
			RobotParts.grabberMotor1.set(0);
			RobotParts.grabberMotor2.set(0);
			RobotParts.grabberMotor3.set(0);
		}
		
		if(control.getRawButton(3))
		{
			RobotParts.extendMotor.set(.3);
		}
		else if(control.getRawButton(2))
		{
			RobotParts.extendMotor.set(-.3);
		}
		else
		{
			RobotParts.extendMotor.set(0);
		}
	}
}

File: frc1678/ManualControls.cpp
#include "ManualControls.h"
#include "WPILib.h"

ManualControls::ManualControls() {
  control_ = new Joystick(1);
}

void ManualControls::Run() {
  // One of the Y motors is backwards
  RobotParts::YAxisMotor1_->Set(-control_->GetY());
  RobotParts::YAxisMotor2_->Set(control_->GetY());

  RobotParts::XAxisMotor_->Set(control_->GetX());

  if(control_->GetRawButton(1)) {
    // One of the grabber motors is backwards
    RobotParts::GrabberMotor1_->Set(1);
    RobotParts::GrabberMotor2_->Set(-1);
    RobotParts::GrabberMotor3_->Set(1);
  } else {
    RobotParts::GrabberMotor1_->Set(0);
    RobotParts::GrabberMotor2_->Set(0);
    RobotParts::GrabberMotor3_->Set(0);
  }

  if(control_->GetRawButton(3)) {
    RobotParts::ExtendMotor_->Set(1);
  }
  else if(control_->GetRawButton(2)) {
    RobotParts::ExtendMotor_->Set(-1);
  } else {
    RobotParts::ExtendMotor_->Set(0);
  }
}
